server/database/build/schema.js

```diff
@@ -0,0 +1,63 @@
+const books = `CREATE TABLE books (
+  id SERIAL PRIMARY KEY,
+  title VARCHAR(255) UNIQUE NOT NULL,
+  author VARCHAR(255) NOT NULL,
+  publish_date DATE,
+  page_count SMALLINT,
+  description TEXT,
+  cover_image VARCHAR(255),
+  price DECIMAL(10, 2) CHECK (price > 0),
+  availability BOOLEAN DEFAULT FALSE
+)`;
+
+const genres = `CREATE TABLE genres ( 
+  id SERIAL PRIMARY KEY,
+  genre VARCHAR(255) UNIQUE NOT NULL
+)`;
+
+const books_genres = `CREATE TABLE books_genres (
+  id SERIAL PRIMARY KEY,
+  book_id INT NOT NULL REFERENCES books,
+  genre_id INT NOT NULL REFERENCES genres
+)`;
+
+const users = `CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  username VARCHAR(255) UNIQUE NOT NULL,
+  password VARCHAR(255) NOT NULL,
+  email VARCHAR(255) UNIQUE NOT NULL,
+  bio JSON
```

Comment:
i think this is really good idea to put bio info as JSON type, smart

Reply:
![image](https://user-images.githubusercontent.com/48598761/222942918-7cf8c38a-3d06-48cd-825b-581202066af4.png)
## tested on other computer, working fine, just need to config .env file correctly
## Good job, Shengfeng!!